### DISCUSSION TITLE
Integration of Pterodactyl with ProxMox

### DISCUSSION THREAD

**Discussion by CodeExplorerX** [pelican-dev/panel#31] [Ideas]
My idea revolves around integrating Pterodactyl with ProxMox to add the capability of managing virtual private servers (VPS) directly from the Pterodactyl interface. This integration would bring several benefits:

1. **Efficient Resource Management:** Optimize CPU, RAM, and disk space usage for each VPS.

2. **Centralized Management:** Manage both game servers and VPS using a single platform.

3. **Flexibility and Scalability:** Easily scale infrastructure according to needs.

4. **Enhanced Security:** Utilize ProxMox's security mechanisms for all virtual servers.

5. **Convenient Monitoring and Tracking:** Easily track status and resource usage through a unified Pterodactyl interface.

I believe this would be a fantastic enhancement. Currently, Pterodactyl offers excellent management features for game servers, but why not expand its functionality by adding VPS management capabilities? This would allow us to manage both game servers and VPS from one place, providing more control over our servers.

However, I understand that some modifications to the management interface may be necessary for VPS. For instance, features like a file manager or direct database management from the control panel might not be needed for VPS. Instead, we could focus on more essential functions such as creating, starting, stopping, and deleting virtual servers, as well as monitoring and resource management.

In any case, I'm confident that the integration of Pterodactyl and ProxMox would bring many benefits.

**Comment by lancepioch** [comment]
You can already create a server with ProxMox and install Wings on it, that's what I actually do! Our integrations are currently directly with Docker and that won't be able to change.
